src/v2/Components/Inquiry/config.ts

```diff
@@ -0,0 +1,179 @@
+import { InquiryInstitutionalAffiliations } from "./views/InquiryInstitutionalAffiliations"
+import { InquirySpecialist } from "./views/InquirySpecialist"
+
+const VIEWS = {
+  Account: InquiryAccount,
+  ArtistsInCollection: InquiryArtistsInCollection,
+  AuctionHousesYouWorkWith: InquiryAuctionHousesYouWorkWith,
+  BasicInfo: InquiryBasicInfo,
+  CommercialInterest: InquiryCommercialInterest,
+  Confirmation: InquiryConfirmation,
+  Done: InquiryDone,
+  FairsYouAttend: InquiryFairsYouAttend,
+  GalleriesYouWorkWith: InquiryGalleriesYouWorkWith,
+  HowCanWeHelp: InquiryHowCanWeHelp,
+  Inquiry: InquiryInquiryQueryRenderer,
+  InstitutionalAffiliations: InquiryInstitutionalAffiliations,
+  Specialist: InquirySpecialist,
+}
+
+type View = keyof typeof VIEWS
+
+export const useEngine = () => {
+  const { user, isLoggedIn } = useSystemContext()
+
+  const engine = useMemo(() => {
+    return new Engine({
+      context: { user, isLoggedIn },
+      workflow: [
+        {
+          askSpecialist: {
+            true: [
+              "Specialist",
+              {
+                isLoggedOut: {
+                  true: ["Account"],
+                },
+              },
+            ],
+            false: [
+              "Inquiry",
+              {
+                isLoggedOut: {
+                  true: ["Account"], // Enforces a logged in user
+                },
+              },
+              {
+                hasCompletedProfile: {
+                  false: ["Confirmation"],
+                },
+              },
+              {
+                hasSeenCommercialInterest: {
+                  false: ["CommercialInterest"],
+                },
+              },
+              {
+                hasBasicInfo: {
+                  false: ["BasicInfo"],
+                },
+              },
+              {
+                isCollector: {
+                  true: [
+                    {
+                      hasSeenArtistsInCollection: {
+                        false: ["ArtistsInCollection"],
+                      },
+                    },
+                    {
+                      hasSeenGalleriesYouWorkWith: {
+                        false: ["GalleriesYouWorkWith"],
+                      },
+                    },
+                    {
+                      hasSeenAuctionHousesYouWorkWith: {
+                        false: ["AuctionHousesYouWorkWith"],
+                      },
+                    },
+                    {
+                      hasSeenFairsYouAttend: {
+                        false: ["FairsYouAttend"],
+                      },
+                    },
+                    {
+                      hasSeenInstitutionalAffiliations: {
+                        false: ["InstitutionalAffiliations"],
+                      },
+                    },
+                  ],
+                },
+              },
+            ],
+          },
+        },
+        "Done",
+      ],
+      conditions: {
```

Comment:
These should really just be inlined with the actual steps for clarity; but I'm just leaving it alone.